frontend/src/Admin/mutations/finalizeShowInvites.graphql

```diff
@@ -0,0 +1,6 @@
+mutation finalizeShowInvites($id: ID! $input: ShowInput!) {
+  updateShow(id: $id, input: $input) {
+    id,
+    finalized
+  }
+}
```

Comment:
nit: new line

If you're using VSCode, might I suggest adding these to your editor's config:

```
"files.trimTrailingWhitespace": true,
"files.insertFinalNewline": true,
"files.trimFinalNewlines": true,
```